R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -17,6 +17,9 @@
 
 context("functions in utils.R")
 
+# Ensure Spark is installed
+sparkCheckInstall()
```

Comment:
What I had in mind was to combine the `sparkR.session` and this `sparkCheckInstall` into one function so its easy to remember for a new test file. Any thoughts on this ?